clarifai/client/app.py

```diff
@@ -288,6 +290,87 @@ def list_concepts(self, page_no: int = None,
       concept_info['id'] = concept_info.pop('concept_id')
       yield Concept(**concept_info)
 
+  def _rich_tree_creation_for_concept_relations(self, relations_dict: Dict[str, Any]) -> None:
+    """Print all the concept relations of the app in rich tree format.
+
+    Args:
+        relations_dict (dict): A dict of concept relations info.
+    """
+    for parent, children in relations_dict.items():
+      tree = Tree(parent)
+      for child in children:
+        tree.add(child)
+      rprint(tree)
+
+  def _concept_relations_accumulation(self, relations_dict: Dict[str, Any], subject_concept: str,
+                                      object_concept: str, predicate: str) -> Dict[str, Any]:
+    """Append the concept relation to relations dict based on its predicate.
+
+    Args:
+        relations_dict (dict): A dict of concept relations info.
+    """
+    if predicate == 'hyponym':
+      if object_concept in relations_dict:
+        relations_dict[object_concept].append(subject_concept)
+      else:
+        relations_dict[object_concept] = [subject_concept]
+    elif predicate == 'hypernym':
+      if subject_concept in relations_dict:
+        relations_dict[subject_concept].append(object_concept)
+      else:
+        relations_dict[subject_concept] = [object_concept]
+    else:
+      relations_dict[object_concept] = []
+      relations_dict[subject_concept] = []
+    return relations_dict
```

Comment:
Can we move this function also to utils folder?

Reply:
Moved to misc.py in utils folder
Done